scripts/ci/pre_commit/pre_commit_check_setup_installation.py

```diff
@@ -81,32 +85,35 @@ def get_extras_from_docs() -> List[str]:
     setup_packages = get_extras_from_setup()
     docs_packages = get_extras_from_docs()
 
-    output_table = ""
+    table = Table()
+    table.add_column("NAME", justify="right", style="cyan")
+    table.add_column("SETUP", justify="center", style="magenta")
+    table.add_column("INSTALLATION", justify="center", style="green")
 
     for extras in sorted(setup_packages.keys()):
         if not set(setup_packages[extras]).intersection(docs_packages):
-            output_table += "| {:20} | {:^10} | {:^10} |\n".format(extras, "V", "")
+            table.add_row(extras, "V", "")
 
     setup_packages_str = str(setup_packages)
     for extras in sorted(docs_packages):
         if f"'{extras}'" not in setup_packages_str:
-            output_table += "| {:20} | {:^10} | {:^10} |\n".format(extras, "", "V")
+            table.add_row(extras, "", "V")
 
-    if output_table == "":
+    if table.row_count == 0:
         sys.exit(0)
 
-    print(
+    rprint(
         f"""
```

Comment:
```suggestion
        f"""\
```
To avoid leading new line - WDYT?

Reply:
done